umbra-js/src/types.ts

```diff
@@ -55,6 +55,19 @@ export interface Announcement {
   token: string;
 }
 
+// Type definition for Announcement event with extra data from the chain
+export interface AnnouncementDetail {
```

Comment:
just curious what the context is around this new type—was it implicitly used before and now just explicitly defined?